.storybook/stories/alertMessage.stories.tsx

```diff
@@ -2,8 +2,8 @@ import React, { FC } from "react"
 
 import { StoryProps } from "./storyProps"
 import StoryContainer from "./storyContainer"
-import AlertMessage, { AlertMessageProps } from "../components/alertMessage"
-import MailSent from "../../.storybook/icons/mailSent"
+import AlertMessage, { AlertMessageProps } from "../../src/components/alertMessage"
```

Comment:
Food for thought - if we import from the index we'd have a bit less pain moving those around. Plus it would fail if the component isn't exported from the package.

Reply:
very good point - I'll move that to another effort though